samples/core/Saving/Saving/Async/Sample.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EFSaving.Async
+{
+    public class Sample
+    {
+        public static void Run()
+        {
+            using (var db = new BloggingContext())
+            {
+                db.Database.EnsureDeleted();
+                db.Database.EnsureCreated();
+            }
+
+            AddBlogAsync("http://sample.com").Wait();
```

Comment:
This isn't shown in the docs... it's just to allow the console app to run the async method. Is there a better way to do this (call an async method from a sync entry point)?